src/status_im2/common/bottom_sheet/style.cljs

```diff
@@ -48,7 +48,7 @@
   [theme window-height sheet-height {:keys [top]}]
   {:position          :absolute
    :bottom            10
-   :max-height        (- window-height sheet-height top)
+   :max-height        (- window-height sheet-height top 10)
```

Comment:
I think this `10` need to be applied for Android only

Reply:
I checked with iOS and it seems necessary too to achieve the same design

Reply:
Also looks a bit like a magic number. Maybe some constant needs to be introduced or at least some explanation?

Reply:
> Also looks a bit like a magic number. Maybe some constant needs to be introduced or at least some explanation?

I have added a comment on the code to explain how we arrive at that value. The selected item take the remaining window height minus (`window-height ` + `bottom-sheet-height` + `(:top insets)`). However, this still does not resolve the margin at the top of the selected-item because the bottom margin eats some of the height of the selected-item